vtrace/breakpoints.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, address, expression=None):
         self.active = False
         self.fastbreak = False      # no NOTIFY_BREAK, autocont, no NOTIFY_CONTINUE
         self.stealthbreak = False   # no NOTIFY_BREAK
+        self.silent = False
```

Comment:
How is silent functionally different than stealthbreak? They seem redundant or potentially bound for odd interaction when both are set?

Reply:
silent doesn't vprint "Hit Break" every time it's hit
stealthbreak doesn't fire notifiers at all.

as i look at this, stealthbreak probably gives me what i'm after.  i want to fastbreak and run bpcode, but not vomit all over the console.  

thoughts?

Reply:
i removed `silent` altogether and allowed Stealthbreak to be toggled per bp instead.